applications/nrf_desktop/src/modules/Kconfig.ble

```
@@ -195,6 +195,15 @@ config DESKTOP_BLE_SCAN_KEYBOARD_LIMIT
 	range 0 255
 	depends on DESKTOP_BLE_SCANNING_ENABLE
 
+config DESKTOP_BLE_SCAN_PM_EVENTS
+	bool "Stop and block scanning in power down mode"
+	default y
+	depends on DESKTOP_BLE_SCANNING_ENABLE
+	depends on CAF_PM_EVENTS
+	help
+	  Stop and block scanning for peers if central switched to power down mode
```

```suggestion
	  Stop and block scanning for peers if the central switched to the power down mode
```
Also "the power down mode" in line 199

---

Fixed both. Thanks!